src/System.ServiceModel.Tests.Common/src/BaseAddress.cs

```diff
@@ -5,6 +5,29 @@
 
 public static class BaseAddress
 {
+#if USE_FIDDLER
+    // Base address for testing nonexistent endpoints
```

Comment:
BTW, my thinking went something like this -- using the new lightweight name-value pair config API's, write an MSBuild task that generates code at build time to populate the name-value pairs directly from available MSBuild properties.  Tests could them poll these name-value pair for things like Bridge URL, Fiddler prefix, etc.